src/mpi/coll/bcast/Makefile.mk

```diff
@@ -8,4 +8,5 @@ mpi_sources +=                     		\
 mpi_core_sources +=											\
 	src/mpi/coll/bcast/bcast_binomial.c						\
 	src/mpi/coll/bcast/bcast_scatter_doubling_allgather.c	\
-	src/mpi/coll/bcast/bcast_scatter_ring_allgather.c		
+    src/mpi/coll/bcast/bcast_scatter_ring_allgather.c		\
+    src/mpi/coll/bcast/bcast_generic_tree.c
```

Comment:
The spacing here is weird. Either the previous spacing is bad or the new spacing is bad, but can it be consistent?